lms/templates/header/header.html

```diff
@@ -29,6 +31,26 @@
     </%block>
 % endif
 
+<%
+  unsupported_browser_alert_versions = configuration_helpers.get_value('UNSUPPORTED_BROWSER_ALERT_VERSIONS', settings.FEATURES.get('UNSUPPORTED_BROWSER_ALERT_VERSIONS'))
+%>
+% if waffle.switch_is_active('enable_unsupported_browser_alert'):
+  <script>
+  var $buoop = {
```

Comment:
buoop is a confusing variable name could we update it to something a little more clear?